Test/Test.cpp

```diff
@@ -122,9 +138,11 @@ int main()
   openaiEndpoint = winrt::OpenAI::builders::OpenAIClient()
 ;
 
-  DoGpt35TurboCompletion();
+//  DoGpt35TurboCompletion();
+
+    DoSimpleChat();
 
-  DoChat();
+//  DoChat();

```

Comment:
Are these supposed to be commented out?

Reply:
Oops, yes that was supposed to be commented out!